### PR TITLE
Add support for all available languages.

### PR DESCRIPTION
**Pull request by jamestalmage**
Performance seems OK to me, maybe a slight lag on keypress (but I could be imagining it, it's really small).

### CODE REVIEW

**Comment by jamestalmage** [comment]
This only requires all the `.js` files. A select few also have `.css` files with them. I probably need to just require those as well.

**Comment by DrewML** [comment]
I believe we can let Browserify handle most of this (instead of a build step + temp file) with [bulkify](https://github.com/substack/bulkify). It should also be able to handle importing the css with the right globbing pattern

**Comment by jamestalmage** [comment]
Indeed. It's been so long since I've done frontend work, I've forgotten so many tricks.

**Comment by DrewML** [comment]
> It's been so long since I've done frontend work

Lucky you 😆

**Comment by jamestalmage** [comment]
Updated to use `bulkify`. It is still not loading some supported languages, and it seems to include some languages that I can't find GFM support for. We probably need to work on a build step that finds the union of those two.

**Comment by jamestalmage** [comment]
List of available languages in gfm: https://raw.githubusercontent.com/github/linguist/master/lib/linguist/languages.yml

This does not match up with what's available in codemirror. Do we want to try and filter to just the union, or defer that until later?

**Comment by DrewML** [comment]
Awesome! Thanks for doing this.

> Do we want to try and filter to just the union, or defer that until later?

Let's just defer for now. Seems like pretty low hanging fruit.

Left one small stylistic comment, but everything looks great besides that.

File: extension/content-script/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import './extension.css';
-import 'codemirror/mode/javascript/javascript';
-import 'codemirror/mode/gfm/gfm';
 import codeMirror from 'codemirror';
 import tabHandler from './tab-handler';
 import onPageNav from './on-page-nav';
 import { removeElement } from './dom';
+var bulk = require('bulk-require');
```
Comment (DrewML) [inline extension/content-script/index.js]: Let's change this to an `import` statement

Comment (jamestalmage) [inline extension/content-script/index.js]: `bulkify` doesn't understand import statements (I tried to initially)

Comment (DrewML) [inline extension/content-script/index.js]: Ahhh, forgot about that limitation - I've hit the same thing with `brfs`. Let's just change it to a `const` (only supporting newer versions of Chrome) and I'll merge it in.

**Comment by jamestalmage** [comment]
done

**Comment by DrewML** [comment]
Thank you @jamestalmage!

**Pull request merged**